nginx.conf

```diff
@@ -97,7 +97,12 @@ http {
         add_header Content-Type text/html;
       }
 
-      location ~ /id_/follow_redirect/(?<proxied_uri>.*) {
+      location ~ ^/id_/follow_redirect/(?<proxied_uri>.*) {
+        # Return HTTP Not Allowed for 127.0.0.0/8 RFC3330 internal loopback block
+        if ( $proxied_uri ~ ^.*//(127\.(([0-9])|([1-9][0-9])|(1[0-9][0-9])|(2(([0-4][0-9])|(5[0-5]))))\.(([0-9])|([1-9][0-9])|(1[0-9][0-9])|(2(([0-4][0-9])|(5[0-5]))))\.(([0-9])|([1-9][0-9])|(1[0-9][0-9])|(2(([0-4][0-9])|(5[0-5]))))).*$ ) {
```

Comment:
I'm not sure how to not duplicate these. I originally was going to abstract the regex into a variable but it seems when I do that it no longer matches and I'm not sure why. I didn't want to spend any more time banging my head against a wall on this so I've left it as is for now.